DUI3-DX/Sdk/Speckle.Autofac/EnumerationExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Speckle.Autofac;
+
+public static class EnumerationExtensions
+{
+  public static void ForEach<T>(this IEnumerable<T> items, Action<T> action)
```

Comment:
Doesn't linq already have effectively this?
https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.foreach?view=netstandard-2.0

Reply:
Ah, its not linq sorry, just `List<T>`
Out of interest, do you really prefer this over just using a regular `ForEach` loop? is this not just slower (due to closure captures)?